feat(Runframe-cli): components

ArnavK-09 opened the pull request:
/claim #88


https://github.com/user-attachments/assets/78eceea0-9214-41f9-ab53-716c5cad32c5

Anshgrover23 reviewed (changes requested):
@ArnavK-09 text is cutting.
![image](https://github.com/user-attachments/assets/a28c6e43-16a6-4139-b2c6-c428d2ad3f81)

seveibar reviewed (changes requested):
let's remove the entrypoint box, it doesn't look good- maybe we'll do it when we support multiple entrypoints

the file menu doesn't look very good, use shadcn as a reference. The export menu is opening in the totally wrong location and shouldn't have a click

Anshgrover23 reviewed (changes requested):
@ArnavK-09 also add the video in desktop as that is easy to understand.

ArnavK-09 commented:
> let's remove the entrypoint box, it doesn't look good- maybe we'll do it when we support multiple entrypoints
> 
> the file menu doesn't look very good, use shadcn as a reference. The export menu is opening in the totally wrong location and shouldn't have a click

Removed entry point box 
Fixed dropdown

Ps - the vid is from phone on desktop it looks clearer and works on hover

https://github.com/user-attachments/assets/b4d0b718-4afb-426c-bfaa-6d70af00d6f4

ArnavK-09 commented:
> @ArnavK-09 text is cutting. ![image](https://private-user-images.githubusercontent.com/168731971/413566880-a28c6e43-16a6-4139-b2c6-c428d2ad3f81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk2NDc0MDYsIm5iZiI6MTczOTY0NzEwNiwicGF0aCI6Ii8xNjg3MzE5NzEvNDEzNTY2ODgwLWEyOGM2ZTQzLTE2YTYtNDEzOS1iMmM2LWM0MjhkMmFkM2Y4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxNVQxOTE4MjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNDg4ZDI1NmY0MGZkMTNkNjAxNTUyYzVmZmUzODA5YzQ1MzAyZDliOWE2MmYxNDM4YjZiODIxN2U0ZGU1MmI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xv4F6KyEVfRXbvHKN0oXHntiaZ71Yd5Bm542SkBk1DY)

its because screen is small, its recording on phone, on deskktop looks great
altho i have added line break that would show full length errors and also will log fully on console

Anshgrover23 reviewed (changes requested):
@ArnavK-09 resolve conflicts.

Anshgrover23 commented:
@ArnavK-09 and thats why i was asking desktop screen video because i want to see more clearly.

ArnavK-09 commented:
> @ArnavK-09 and thats why i was asking desktop screen video because i want to see more clearly.

It's working fine in desktop, you can check, I can't send desktop recording i don't have medium

seveibar commented:
it does look a lot better. I think we might want to use a smaller font size for everything in that menu though, either sm or xs, it's just too big for a file menu imo

ArnavK-09 commented:
> it does look a lot better. I think we might want to use a smaller font size for everything in that menu though, either sm or xs, it's just too big for a file menu imo

fixed font size

seveibar commented on lib/components/RunFrameForCli/LeftHeader.tsx:
```diff
@@ -0,0 +1,212 @@
+                        event_type: "REQUEST_EXPORT",
+                        exportType: ext,
+                      })
+                      setNotificationMessage(`Export requested: ${ext}`)
```
this is good, fwiw on the cli we will detect this and automatically trigger a download in the browser by hitting the files api, so from the user's perspective they'll just get a file download

To do that properly you would use `window.open` to the appropriate files api here, can be done in a future PR

seveibar commented:
@ArnavK-09 fwiw i can't see the reduced font size because browser preview doesn't work for the Cli header rn, something to look at in a future PR, for now trust
